base-hack/src/misc/win_condition.c

```diff
@@ -142,8 +142,51 @@ int canAccessKroolsChallenge(void) {
     return 1;
 }
 
+int canAccessWinCondition(void) {
```

Comment:
For code opt purposes, call this function in checkSeedVictory so we have 1 source of truth with what triggers a win con